specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "skipToken",
+            "in": "query",
+            "description": "Skip token to continue getting comments from a given point",
+            "x-nullable": true
+          },
+          {
+            "type": "integer",
+            "name": "pageSize",
+            "in": "query",
+            "description": "maximum number of comments to return",
+            "format": "int32",
+            "default": 50,
+            "x-nullable": false
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "default": "2023-01-01-preview",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of comments for the given review",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewCommentDto"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "description": "In case request structure is wrong",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "500": {
+            "x-nullable": false,
+            "description": "In case an error has occurred in the server",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PageResultOfReviewDto": {
+      "type": "object",
+      "required": [
+        "count"
+      ],
+      "properties": {
+        "items": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReviewDto"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "nextPageLink": {
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "ReviewDto": {
+      "type": "object",
+      "required": [
+        "rating",
+        "createdDateTime",
+        "lastModifiedDateTime",
+        "commentsCount",
+        "helpfulCount"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "uniqueProductId": {
+          "type": "string"
+        },
+        "rating": {
+          "type": "number",
+          "format": "decimal"
+        },
+        "title": {
+          "type": "string"
+        },
+        "content": {
+          "type": "string"
+        },
+        "source": {
+          "type": "string"
+        },
+        "reviewerName": {
+          "type": "string"
+        },
+        "externalReviewUrl": {
+          "type": "string"
+        },
+        "isvReply": {
```

Comment:
Is the `v` a typo and should it be `isReply`?

Reply:
Or is this ISV = Independent Software Vendor? I am not sure if we want to use that term (seems more like how we as a business categorize our customers into segments) in the API. Maybe `developerReply` or `appAuthorReply`?

Reply:
Ted's suggestion: Product owner instead of ISV. But check with existing language in marketplace documentation.